ArmyOfOnes/ArmyOfOnes/ViewControllers/CurrenciesListViewController.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  CurrenciesListViewController.swift
+//  ArmyOfOnes
+//
+//  Created by Fabian Mariño on 11/8/16.
+//  Copyright © 2016 Fabian Mariño. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+enum LayoutConstraints: CGFloat {
+    case inputOffset = 14,
+    topInset = 43,
+    tableTopInset = 70
+}
+
+enum textFieldLimit: Int {
+    case max = 15
+}
+
+class CurrenciesListViewController: UITableViewController {
+    
+    var didSetupConstraints = false
+    var dollarQuantity = 1.0
+    
+    let quantityInput: UITextField = {
+        let textField = UITextField()
+        textField.text = String(1.0)
+        textField.placeholder = "Type a valid dollar quantity"
+        textField.layer.cornerRadius = 4.0
+        textField.layer.masksToBounds = true
+        textField.layer.borderColor = UIColor.lightGray.cgColor
+        textField.layer.borderWidth = 1.0
+        textField.textAlignment = .center
+        textField.keyboardType = UIKeyboardType.decimalPad
+        return textField
+    }()
+    
+    let cellIdentifier = "CellIdentifier"
+    let listTitle = "Army of Ones"
+    
+    var currencyRatesList = [String]()
+    var currencyCountries = [String]()
+    var currencyFormatter = NumberFormatter()
+    
+    var rates = [Currency]()
+    
+    //Mark: View methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.contentInset = UIEdgeInsetsMake(LayoutConstraints.tableTopInset.rawValue, 0, 0, 0);
+        tableView.isScrollEnabled = false
+        tableView.allowsSelection = false
+        tableView.separatorStyle = UITableViewCellSeparatorStyle.none
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        title = listTitle
+        tableView?.register(UITableViewCell.self, forCellReuseIdentifier: cellIdentifier)
+        
+        // Gesture that recognices tap to hide keyboard
+        let tapGesture = UITapGestureRecognizer(target: self, action:#selector(CurrenciesListViewController.endEditing(_:)))
+        tapGesture.cancelsTouchesInView = false
+        tableView.addGestureRecognizer(tapGesture);
```

Comment:
You can group the table view configuration in a dedicated method like `tableViewConfig` or `tableViewSetu`. Whatever method name you see fit.